packages/dht/src/dht/routing/RoutingSession.ts

```diff
@@ -90,7 +90,7 @@ export class RoutingSession extends EventEmitter<RoutingSessionEvents> {
         const previousId = previousPeer ? getNodeIdFromPeerDescriptor(previousPeer) : undefined
         this.contactList = new SortedContactList({
             referenceId: getDhtAddressFromRaw(config.routedMessage.target),
-            maxSize: 10000,  // TODO use config option or named constant?
+            maxSize: 10,  // TODO use config option or named constant?
```

Comment:
Do we have a some config option for `DhtNode` which correlates to this value? E.g. would this be slightly greater than `numberOfNodesPerKBucket`, or something like that? (If that is the case, maybe we derive the value from that option.)

Reply:
yeah, a name would be good. There's no need to have this be configurable. It's just more of a sanity limit for the structure. There's no need to keep a sorted list of 10000 connections when we only want to know the closest few. The limit could be even lower than 10 because of this...

perhaps something like `ROUTING_CANDIDATE_LIMIT`